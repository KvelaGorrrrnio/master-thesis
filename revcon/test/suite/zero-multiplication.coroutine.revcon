@coroutine

coroutine main() {
  local int a 10
  a *= 0
  delocal int a
}
