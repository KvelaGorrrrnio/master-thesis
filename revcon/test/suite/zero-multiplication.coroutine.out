An attempt to destructively make a zero multiplication in a *= 0.
